src/adapter/src/config/frontend.rs

```diff
@@ -33,17 +35,48 @@ pub struct SystemParameterFrontend {
 impl SystemParameterFrontend {
     /// Construct a new [SystemParameterFrontend] instance.
     pub fn new(
+        env_id: EnvironmentId,
         ld_sdk_key: &str,
-        ld_user_key: &str,
         ld_key_map: BTreeMap<String, String>,
     ) -> Result<Self, anyhow::Error> {
         let ld_config = ld::ConfigBuilder::new(ld_sdk_key).build();
         let ld_client = ld::Client::build(ld_config)?;
-        let ld_user = ld::User::with_key(ld_user_key).build();
+        let ld_ctx = if env_id.cloud_provider() != "local" {
+            // We don't use the Display for EnvironmentId implementation here
+            // because we don't want to have the ordinal as part of the LD key.
```

Comment:
Wait, sorry, maybe I was confusing on Slack, but I'm almost positive we _do_ want to include the ordinal in the key. If we ever support multiple environments per regions, they'll be framed as separate "spaces" that would be subject to different limits and feature flags.

Reply:
Ah, yes, they'll definitely share the same `MZ_ENVIRONMENT_ID` value. As in, it's the "environment ID", not the "environmentd ID". Very load bearing "d" there. If we ever have multiple environmentds in a single environment, we'll invent some new "environmentd ID" for sure!